personio-personnel-data-api-oa3.yaml

```diff
@@ -374,51 +411,6 @@ paths:
                 "employee[weekly_hours]":
                   description: Employee weekly working hours
                   type: number
-          application/json:
-            schema:
-              type: object
-              properties:
-                employee:
-                  type: object
-                  properties:
-                    first_name:
-                      type: string
-                      example: "John"
-                    last_name:
-                      type: string
-                      example: "Dou"
-                    custom_attributes:
-                      type: object
-                      properties:
-                        gender:
-                          type: string
-                          example: male
-                        position:
-                          type: string
-                          example: developer
-                        subcompany:
-                          type: string
-                          description: The subcompany employee belongs to. Should be predefined in Personio. Otherwise will be ignored with showing meta error in the response.
-                          example: ACME
-                        department:
-                          type: string
-                          description: The department employee belongs to. Should be predefined in Personio. Otherwise will be ignored with showing meta error in the response.
-                          example: IT
-                        office:
-                          type: string
-                          description: The office employee belongs to. Should be predefined in Personio. Otherwise will be ignored with showing meta error in the response.
-                          example: Madrid
-                        hire_date:
-                          type: string
-                          example: '2020-01-31'
-                          format: yyyy-mm-dd
-                        weekly_hours:
-                          type: number
-                          example: 40
-                        dynamic_{{ field uid }}:
-                          type: string
-                          description: Dynamic field, represented by unique id.
-                          example: "German"
       responses:
         "200":
           description: Employee is updated
```

Comment:
The employee created response doesn't have `message` attribute, I tried again the API and the output is

```json
{
  "success": true,
  "data": {
    "id": 2592937
  },
  "meta": []
}
```